demo/Dockerfile

```diff
@@ -1,25 +1,27 @@
-FROM jenkins:2.7.1
+FROM jenkins:2.7.3
 
 USER root
-ENV MAVEN_VERSION 3.3.9
-RUN cd /usr/local; wget -O - http://mirrors.ibiblio.org/apache/maven/maven-3/$MAVEN_VERSION/binaries/apache-maven-$MAVEN_VERSION-bin.tar.gz | tar xvzf -
-RUN ln -sv /usr/local/apache-maven-$MAVEN_VERSION/bin/mvn /usr/local/bin
-USER jenkins
 
-WORKDIR /tmp
-ADD repo /tmp/repo-wc-orig
-RUN cp -rv repo-wc-orig repo-wc
+RUN wget -q -O /usr/bin/docker --no-check-certificate https://get.docker.com/builds/Linux/x86_64/docker-1.9.0 && \
```

Comment:
I do not want the whole Docker package, just the client executable. In fact I am considering deleting even this and just running with (found this tip somewhere on the Internet):

    -v $(which docker):/usr/bin/docker